Controller/TaskController.php

```diff
@@ -354,6 +354,7 @@ public function putAction(string $id, Request $request): Response
         $task->setSchedule($dateTime);
         $task = $this->taskManager->update($task);
 
+        $this->entityManager->merge($task);
```

Comment:
Not sure about this but seems currently the only solution and TargetGroup was handled the same way.